patches/source-map-support+0.5.16.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/source-map-support/source-map-support.js b/node_modules/source-map-support/source-map-support.js
```

Comment:
where did yo get this from? no documentation (either here or where its applied)

Reply:
I will DM you on this